dashboard/src/actions/overviewActions.js

```diff
@@ -49,9 +49,7 @@ export const getDatasets = () => async (dispatch, getState) => {
       clearCachedSession(dispatch);
     } else {
       const msg = error.response?.data?.message;
-      dispatch(
-        showToast(DANGER, msg ? msg : `Error response: ERROR_MSG`)
-      );
+      dispatch(showToast(DANGER, msg ? msg : `Error response: ERROR_MSG`));
```

Comment:
No need to change this unless you have other changes to make, but a recent change from Varshini brought to light the fact that we have [alternatives](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#description) to the use of the ternary operator, here.  For instance, either `||` or `??` might be nicer.
```javascript
      dispatch(showToast(DANGER, msg || `Error response: ERROR_MSG`));
```
(`??` will use the default message if `msg` is `undefined`; `||` will use the default message if `msg` is an empty string; but using either of them allows us to avoid the second reference to `msg`...just some more items for our Javascript toolboxes....)

Reply:
Okay, I don't have any blocking changes outstanding in this PR now so I'll defer it in another PR.